xarray/core/dataset.py

```diff
@@ -3934,9 +3935,23 @@ def merge(
             - 'left': use indexes from ``self``
             - 'right': use indexes from ``other``
             - 'exact': error instead of aligning non-equal indexes
+
         fill_value : scalar or dict-like, optional
             Value to use for newly missing values. If a dict-like, maps
             variable names (including coordinates) to fill values.
+        combine_attrs : {"drop", "identical", "no_conflicts", "drop_conflicts", \
+                        "override"}, default: "override"
+            String indicating how to combine attrs of the objects being merged:
+
+            - "drop": empty attrs on returned Dataset.
```

Comment:
```suggestion
            - "drop": empty attrs on the returned Dataset.
```

Reply:
that was copied from `merge` and should be the same for all functions / methods that accept `combine_attrs` so I would change that in a new PR.